src/overlays/actors/ovl_En_Fz/z_en_fz.c

```diff
@@ -863,14 +863,14 @@ void EnFz_Draw(Actor* thisx, PlayState* play) {
 
     if (this->drawDmgEffTimer > 0) {
         s32 pad2[6];
-        Vec3f limbPos[2];
+        Vec3f bodyPartsPos[2];
```

Comment:
I can see it not being worth it, but should this also get an enum?

Reply:
Yeah, I wasn't sure about this. It would be an enum local to this function which has been done before, but idk